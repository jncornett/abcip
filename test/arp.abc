d ( stack="eth:arp" )
a ( sip="192.168.1.1", tip="192.168.2.2" )

