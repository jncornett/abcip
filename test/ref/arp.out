ARP, Ethernet (len 6), IPv4 (len 4), Request who-has 192.168.2.2 (02:09:08:07:06:05) tell 192.168.1.1, length 28
	0x0000:  0209 0807 0605 0201 0203 0405 0806 0001  ................
	0x0010:  0800 0604 0001 0201 0203 0405 c0a8 0101  ................
	0x0020:  0209 0807 0605 c0a8 0202                 ..........
